packages/common/src/scorekeeper/jobs/MicroserviceJobRunner.ts

```diff
@@ -1,83 +0,0 @@
-import { logger } from "../..//index";
```

Comment:
So, the idea is to nuke everything related to the Microservice version, right ?

Reply:
Yes, but the idea of decoupling from the Gateway is still relevant. Implementing with `bullmq` is one option. BTW even with the queue implementation, I see no usage for that class. All the jobs would be initialized on the worker side and the worker becomes the new Scorekeeper in that case.

However, a simpler approach would be to leave everything as is and only change the job status handling. By persisting the job status in the DB and removing events, we can achieve similar results and decouple Scorekeeper from API. That's what I suggest.

Moving from this:
`await createServer(config, handler, scorekeeper);`
To this:
`await createServer(config, handler);`

This way API could get the job statuses directly from the DB and there would be no need to run everything tightly coupled.